projects/worker-animal-pictures/README.md

```diff
@@ -0,0 +1,31 @@
+# worker-animal-pictures
+
+Example of a 3rd party Asset Compute worker. `worker-animal-pictures` produces renditions from an HTTP request to an external API based on an incoming parameter.
+
+It is based on [Project Firefly](https://github.com/AdobeDocs/project-firefly) and the [aio](https://github.com/adobe/aio-cli) developer tool.
+
+## Install
+
+Requirements:
+
+* [aio cli](https://github.com/adobe/aio-cli)
+
+### Test
+
+This executes eslint and the worker test cases located in [tests](tests):
+
+```
+npm test
```

Comment:
Should probably use aio app test instead, to be in line with Firefly and our docs.

Reply:
what about listing commands for [`deploy`](https://git.corp.adobe.com/nui/nui/blob/master/doc/developer/CustomWorkerDeveloperGuide.md#deploy), [`run`](https://git.corp.adobe.com/nui/nui/blob/master/doc/developer/CustomWorkerDeveloperGuide.md#running-the-application), and [debugging](https://git.corp.adobe.com/nui/nui/blob/master/doc/developer/CustomWorkerDeveloperGuide.md#debug)?

Reply:
(We could say aio is to be used, so commands they may add will be covered too).